### DISCUSSION TITLE
landingpage

### DISCUSSION THREAD

**Discussion by YudhaPratamadya** [imamsudarajat04/nongkrongkoding#44] [General]
![Annotation 2022-12-16 191811](https://user-images.githubusercontent.com/107564194/208096982-f05f6e4e-c6dd-4cc5-ae08-c07ee14c7521.png)
ini icon nya mau di ganti gimana gays?

**Comment by imamsudarajat04** [comment]
![image](https://user-images.githubusercontent.com/70099731/208257127-bd6a90d3-15ac-4ccb-a806-e78614bc8063.png)
Information Security didepan yud, kali linux belakang

Bagian Frontend dan backend GitHub nya diganti atau di buang

**Comment by imamsudarajat04** [comment]
![image](https://user-images.githubusercontent.com/70099731/208257215-24d6ef32-2aac-48e6-9201-27fb1c059e95.png)
Kata2nya kelewatan, cari css atau js yang bisa kaya gini "Lorem Ipsum Asddsaa ......" jadi dia gak kelewatan

**Comment by imamsudarajat04** [comment]
![image](https://user-images.githubusercontent.com/70099731/208257246-7ef0f3aa-bfd0-4b29-bc24-701d00b7677e.png)
Ganti ke bahasa indonesia
